pkl-core/src/main/java/org/pkl/core/EvaluatorImpl.java

```diff
@@ -157,6 +157,15 @@ public Object evaluateOutputValue(ModuleSource moduleSource) {
         });
   }
 
+  private VmTyped readModuleOutput(VmTyped module) {
+    var value = VmUtils.readMember(module, Identifier.OUTPUT);
+    if (value instanceof VmTyped typedOutput && typedOutput.getVmClass().getPClassInfo() == PClassInfo.ModuleOutput) {
+      return typedOutput;
+    }
```

Comment:
Sorry, my suggestion needs reformatting:

```suggestion
    if (value instanceof VmTyped typedOutput
        && typedOutput.getVmClass().getPClassInfo() == PClassInfo.ModuleOutput) {
      return typedOutput;
    }
```